src/locales/translations/de.ts

```diff
@@ -0,0 +1,79 @@
+export default {
+  favorites: "Favoriten",
+  internet: "Internet",
+  partners: "Partner",
+  recently_joined: "Kürzlich beigetreten",
+  nickname: "Nickname",
+  settings: "Eintstellungen",
+  minimize: "Minimieren",
+  maximize: "Maximieren",
+  close: "Schließen",
+  add_server_modal_description_1: "Server manuell zu Favoriten hinzufügen.",
+  add_server_modal_description_2: "Beispiel: 127.0.0.1:7777",
+  add: "Hinzufügen",
+  server: "Server",
+  address: "Adresse",
+  players: "Spieler",
+  server_join_prompt_enter_password:
+    "Dieser Server ist passwortgeschützt, bitte gib ein Passwort ein.",
+  server_join_prompt_enter_password_input_placeholder: "Passwort eingeben...",
+  server_join_prompt_nickname_input_placeholder: "Nickname eingeben...",
+  connect: "Verbinden",
+  copy: "Kopieren",
+  remove_from_favorites: "Aus Favoriten entfernen",
+  add_to_favorites: "Zu Favoriten hinzufügen",
+  filters: "Filter",
+  filter_only_omp_servers: "Nur open.mp Server",
+  filter_non_empty_servers: "Server mit Spielern",
+  filter_unpassworded_servers: "Server die nicht passwortgeschützt sind",
+  rule: "Regel",
+  value: "Wert",
+  player: "Spieler",
+  score: "Punkte",
+  locked: "Gesperrt",
+  unlocked: "Entsperrt",
+  openmp_server: "open.mp Server",
+  name: "Name",
+  ping: "Ping",
+  mode: "Modus",
+  filter_servers: "Server filtern",
+  search_for_server_hostname_mode: "Anhand von Hostname oder Modus nach Server suchen",
```

Comment:
```suggestion
  search_for_server_hostname_mode: "Nach Server-Hostname/Modus suchen",
```

Reply:
IS this even search btw, or is it filtering? Search jumps, filter reducs.